### PR TITLE
docs(vscode): expand vscode extension docs

### PR DESCRIPTION
**Pull request by robertohuertasm**
### What does this PR do? What is the motivation?

Harmonizes the current documentation and the extension documentation used in the [Microsoft marketplace](https://marketplace.visualstudio.com/items?itemName=Datadog.datadog-vscode)

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

Related to https://github.com/DataDog/datadog-vscode/pull/1449

IDE-2463
IDE-2916

### CODE REVIEW

File: content/en/developers/ide_plugins/vscode/_index.md [hunk 1]
```diff
@@ -91,44 +101,105 @@ The **View in VS Code** feature provides a link from Datadog directly to your so
...
-To share your feedback, email [team-ide-integration@datadoghq.com][9].
+Check out the [issues][known_issues] section to discover known issues.
 
 ## Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: /continuous_testing/
-[2]: https://www.datadoghq.com/
+[datadog]: https://www.datadoghq.com/
```
Comment (rtrieu) [inline content/en/developers/ide_plugins/vscode/_index.md:180]:
```suggestion
[datadog]: https://www.datadoghq.com/
```

Hi @robertohuertasm, can you please update these relative link references to the correct consecutive numbers to be consistent with the docs style guide? Thank you!

**Review by rtrieu — CHANGES_REQUESTED** [review]
Left some feedback for clarity and consistency, but otherwise looks good!

**Comment by robertohuertasm** [comment]
Hi @rtrieu, thank you very much for your review! I think I have addressed all the issues. Let me know if you find something else.

File: content/en/developers/ide_plugins/vscode/_index.md [hunk 2]
```diff
@@ -91,27 +101,74 @@ The **View in VS Code** feature provides a link from Datadog directly to your so
...
-## Feedback
+## License
+
+Please read this [End-User License Agreement][23] carefully before downloading or using the Datadog Visual Studio Code Extension.
+
+## Data and telemetry
+
+Datadog anonymously collects information about your usage of this IDE, including how you interact with it, whether errors occurred while using it, and what caused those errors, in accordance with the [Datadog Privacy Policy][24] and Datadog's [VS Code extension EULA][23].
+
+If you don't wish to send this data to [Datadog][2], you can opt out at any time in the extension settings: `Datadog > Telemetry > Setup > Enable Telemetry`.
```
Comment (rtrieu) [inline content/en/developers/ide_plugins/vscode/_index.md:163]:
```suggestion
If you don't wish to send this data to [Datadog][2], you can opt out at any time in the VS Code extension settings: `Datadog > Telemetry > Setup > Enable Telemetry` and select `disabled`.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
A few more things!

**Comment by robertohuertasm** [comment]
@rtrieu committed all your suggestions! Thanks a lot!

**Review by rtrieu — APPROVED** [review]
It looks great! Thank you!

**Pull request merged**